docs/en/Running-Spark-on-Alluxio.md

```diff
@@ -177,17 +177,16 @@ spark.sql.hive.metastore.sharedPrefixes=com.mysql.jdbc,org.postgresql,com.micros
 
 If the recommended solution described above is infeasible, this is a workaround which can also solve this issue.
 
-Specifying the Hadoop configuration `fs.alluxio.impl` may also help in resolving this error. `fs.alluxio.impl` should
-be set to `alluxio.hadoop.FileSystem` and if you are using Alluxio in fault tolerant mode, `fs.alluxio-ft.impl` should
-be set to `alluxio.hadoop.FaultTolerantFileSystem`. There are a few alternatives to set these parameters.
+Specifying the Hadoop configuration `fs.alluxio.impl` may also help in resolving this error.
+`fs.alluxio.impl` should be set to `alluxio.hadoop.FileSystem`. There are a few alternatives to set
```

Comment:
might be clearer to change "alternatives" to "ways". Atm it sounds like the parameters might not need to be set since there are alternatives to setting them.